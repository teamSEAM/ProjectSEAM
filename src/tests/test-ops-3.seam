# TEST NEGATIVE NUMBERS #
entity World:
	func start():
		if((1+(-2)==-1):
			screen.out("Addition PASS")
		else:
			screen.out("Addition FAIL")

		if((3-(-2)==5):
			screen.out("Subtraction PASS")
		else:
			screen.out("Subtraction FAIL")

		if((3*-2)==-6):
			screen.out("Multiplication PASS")
		else:
			screen.out("Multiplication FAIL")

		if((3/-3)==-1):
			screen.out("Division PASS")
		else:
			screen.out("Division FAIL")


