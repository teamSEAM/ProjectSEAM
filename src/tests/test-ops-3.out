Addition PASS
Subtraction PASS
Multiplication PASS
Division PASS
